src/main/resources/graphiql.html

```diff
@@ -37,8 +37,8 @@
       copy them directly into your environment, or perhaps include them in your
       favored resource bundler.
      -->
-    <link rel="stylesheet" href="//cdn.jsdelivr.net/graphiql/0.8.0/graphiql.css" />
-    <script src="//cdn.jsdelivr.net/graphiql/0.8.0/graphiql.min.js"></script>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/graphiql/0.9.3/graphiql.css" />
+    <script src="//cdn.jsdelivr.net/graphiql/0.9.3/graphiql.min.js"></script>
```

Comment:
Sweet, done in https://github.com/astorije/beer/commit/762db15568a4e1ffc5d81056d9ca3008efc54575. Thanks!